newIDE/app/src/ObjectsList/index.js

```diff
@@ -228,7 +231,8 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
         if (onEditObject) {
           onEditObject(object);
           onObjectCreated(object);
-          onObjectSelected(name);
+          // Adding an object is always (at the moment) going to the scene (layout), and not to the project (as a global object) so the context is not global.
+          onObjectSelected({ object, global: false }); // objectWithContext
```

Comment:
I think the usual way to mention the name of the parameter is to use 
```suggestion
          onObjectSelected(/* objectWithContext */{ object, global: false });
```